src/components/Banners/UpdateMessage.jsx

```diff
@@ -0,0 +1,43 @@
+/* global cozy */
+
+import React, { Component } from 'react'
+import { translate } from 'cozy-ui/react/I18n'
+
+import Banner from 'components/Banners/Banner'
+
+export class UpdateMessage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isRedirecting: false
+    }
+    this.redirectToStore = this.redirectToStore.bind(this)
+  }
+
+  async redirectToStore() {
+    this.setState({ isRedirecting: true })
+    const { konnector } = this.props
+    await cozy.client.intents.redirect('io.cozy.apps', {
+      slug: konnector.slug,
+      step: 'update'
+    })
```

Comment:
No try / catch ? (I don't know the behavior of `intents.redirect` so maybe it's not needed)

Reply:
It was just a copy/paste but you're right. I add it 👍